ui/menus.md

```diff
@@ -0,0 +1,58 @@
+### Menus
+
+The menu system allow definition of multiple menus. Currently we use 2:
+
+* Main menu
+* Top right menu
+
+Each menu consists of menu sections and menu items. Menu sections can contain other menu sections and menu items.
+
+The main menu styling supports 3 level, that is 2 levels of menu section. Menus are defined in ``app/presenters/menu/default_menu.rb``
+
+#### Menu Sections
+
+```
+Menu::Section.new(:clo, N_("Clouds"), 'fa fa-plus fa-2x', [
+```
+
+Menu sections looks like the example above. It consists of: 
```

Comment:
Clarity, but maybe tie each of these items to the definitions that follow, either with numbers or maybe **bolding** the leading words?  

> Menu sections, such as the example above consist of the following items: 
> 1.  Identifier
> 2.  Text
> 3.  Icon 
> 4.  Items